features/admin_features/admin_adds_new_user.feature

```diff
@@ -5,10 +5,66 @@ Feature: Admin can administer users
 
   Background:
     Given An admin exists "admin@example.com" "password"
+    Given An admin exists "restaurant_owner@example.com" "password" "false"
```

Comment:
Okay, so I do understand what is happening but this but this step should, imo, be more clear. Imaging a non-technical stakeholder reading this?

Reply:
yes, we were thinking the same just hadn't gotten around to change it.